src/Core/src/Core/IWindow.cs

```diff
@@ -130,6 +130,7 @@ public interface IWindow : ITitledElement
 		float RequestDisplayDensity();
 
 #if WINDOWS
+		ITitleBar? TitleBar => null;
 		Rect[]? TitleBarDragRectangles => null;
```

Comment:
What happens if I set `TitleBar` and the `TitleBarDragRectangles`? Will they conflict? Should we mark it as obsolete and say that TitleBar overrides the rects?

Reply:
Yes, that's a good idea

Reply:
Oh wait, JK -- this is not exposed publicly